src/client/push-client.js

```diff
@@ -105,7 +105,9 @@ export default class PushClient extends EventDispatch {
 
   _dispatchStatusUpdate() {
     return Promise.all([
-      this.getSubscription(),
+      this.getSubscription().catch(() => {
+        return null;
```

Comment:
There is no subscription and if we don't catch the error the Promise.all fails and the event is not dispatched. This is largely the driving factor behind: https://github.com/GoogleChrome/Propel/issues/66